static/js/home.js

```diff
@@ -74,18 +75,34 @@ $(document).ready( function(){
   }
 
   var generate_new_path = function(selected) {
-    var new_parent = GLOBALS.current_parent
+    new_parent = GLOBALS.current_parent
```

Comment:
Don't remove `var`. The variable will become a global variable if you remove it. Using `var` will make it a local one.